wdl/ONTFlowcellFromMultipleBasecalls.wdl

```diff
@@ -0,0 +1,85 @@
+version 1.0
+
+import "tasks/Utils.wdl" as Utils
+import "tasks/Finalize.wdl" as FF
+
+import "tasks/SampleLevelAlignedMetrics.wdl" as COV
+
+workflow ONTFlowcellFromMultipleBasecalls {
+    input {
+        Array[File] aligned_bams
+        Array[File] aligned_bais
+        Boolean bams_suspected_to_contain_dup_record = true
+
+        String flowcell
+
+        File? bed_to_compute_coverage
+
+        File ref_map_file
+
+        String gcs_out_root_dir
+    }
+
+    parameter_meta {
+        bams_suspected_to_contain_dup_record: "if the multiple basecalls are suspected to have duplicates amongst them"
+    }
+
+    Map[String, String] ref_map = read_map(ref_map_file)
+
+    String outdir = sub(gcs_out_root_dir, "/$", "") + "/ONTFlowcell/~{flowcell}"
```

Comment:
Are you sure output is meant to go into the usual ONTFlowcell directory? I'm fine with that, but just wanted to check that this is what you intended.

Reply:
yes, that's intended. the multiple basecalls themselves are output to a different dir

https://github.com/broadinstitute/long-read-pipelines/blob/a7527a5e51ae3710f78ee48af28ed97c60d5801d/wdl/ONTBasecall.wdl#L15

It's done this way so that flowcell level data are all stored in `ONTFlowcell`.

One disparity between 1-basecall flowcells and 1+-basecall flowcells is that the latter has many metrics stored in the basecalls directory, and how they should be aggregated to the flowcell level is to be done in the planned overhaul.

Thoughts?